lib/python/gladevcp/hal_python.xml

```diff
@@ -63,7 +62,7 @@
         <glade-widget-class name="HAL_VBar" generic-name="hal_vbar" title="HAL VBar"/>
         <glade-widget-class name="Override" generic-name="override" title="Override Controls">
             <properties>
-                <property id="Adjustment" visible="false"/>
+                <property id="Adjustment" visible="False"/>
```

Comment:
because diff'ing glade files doesn't produce useful information, such changes are not useful to review.